zulipterminal/model.py

```diff
@@ -567,8 +567,14 @@ def update_stream_message(
         if response["result"] == "success":
             old_topic = self.index["messages"][message_id].get("subject", None)
             new_topic = request["topic"]
+            stream_name = self.index["messages"][message_id].get(
+                "display_recipient", None
+            )
```

Comment:
Let's simplify using a well-named variable for the common code here.

Reply:
get_message = self.index["messages'] for common code.